src/sbvr-api/permissions.coffee

```diff
@@ -122,7 +123,28 @@ exports.setup = (app, sbvrUtils) ->
 		PREPARSE: ({ req }) ->
 			apiKeyMiddleware(req)
 		POSTPARSE: ({ req, request }) ->
+			if (request.method == 'POST' and request.odataQuery.property?.resource == 'canAccess')
+				request.odataQuery.options = { '$select': { 'properties': [ { 'name': 'id' } ] } }
+				{ action, method } = request.values
+				if method? and methodPermissions[method]?
+					request.permissionType = methodPermissions[method]
+				else if action?
+					request.permissionType = action
+				else
+					throw new MissingOrInvalidArgumentsError()
+				request.resourceName = request.resourceName.slice(0, request.resourceName.length - '#canAccess'.length)
+				request.odataQuery.resource = request.resourceName
+				delete request.odataQuery.property
+				request.method = 'GET'
```

Comment:
I guess we are changing the method on the request for the subsequent addPermissions call. What about calling `_addPermission` directly with `methodPermissions.GET`? This way we could avoid mutation on `request`

Reply:
No we change the method to perform only a read only select, because an action is always called as `POST`. The permissions are collected by setting `request.permissionType`.

Reply:
Ohhh, that makes sense. I guess mutating it makes sense then